src/installer/pkg/sfx/Directory.Build.props

```diff
@@ -0,0 +1,25 @@
+<Project>
+  <PropertyGroup>
+    <EnsureRuntimeIdentifierSet>true</EnsureRuntimeIdentifierSet>
+  </PropertyGroup>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props, $(MSBuildThisFileDirectory)..))" />
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <!-- The built-in packaging targets make PackageLicenseFile and PackageLicenseExpression
+         mututally exclusive. Use PackageLicenseExpression if it's available. -->
+    <PackageLicenseFile Condition="'$(PackageLicenseExpression)' != ''"></PackageLicenseFile>
+    <PackageThirdPartyNoticesFile>$(RepoRoot)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
+    <!-- This link should be updated for each release milestone, currently this points to 1.0.0 -->
+    <ReleaseNotes>https://go.microsoft.com/fwlink/?LinkID=799417</ReleaseNotes>
+    <ProjectUrl>https://dot.net</ProjectUrl>
+    <Authors>Microsoft</Authors>
+    <MinClientVersion>2.12</MinClientVersion>
+    <LicenseFile>$(RepoRoot)LICENSE.TXT</LicenseFile>
+    <ProductBrandPrefix>Microsoft .NET</ProductBrandPrefix>
```

Comment:
Should these properties be centrally defined? We set those in libraries as well.

Reply:
I think we can move these up and share them in a future PR.

Reply:
Opened https://github.com/dotnet/runtime/issues/44536 to track this.